lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -333,6 +334,58 @@ def _pixelsize(self):
         """Kymograph (spatial) pixel size in physical units."""
         return self._kymo.pixelsize[0]
 
+    def _model_fit(self, node_idx):
+        """Extract fitted model from localization model at particular node if available"""
+        num_coords = len(self)
+        node_idx = num_coords + node_idx if node_idx < 0 else node_idx
+        if not 0 <= node_idx < num_coords:
+            raise IndexError(
+                f"Node index is out of range of the KymoTrack. Kymotrack has length {num_coords}."
+            )
+
+        pixelsize, num_pixels = self._kymo.pixelsize[0], self._kymo.pixels_per_line
+        coords = np.arange(0.0, num_pixels * pixelsize, 0.1 * pixelsize)
+        model = self._localization.evaluate(coords, node_idx, pixelsize)
+
+        return coords, model
+
+    def plot_fit(self, node_idx, *, fit_kwargs=None, data_kwargs=None, show_data=True):
+        """Plot the localization model
+
+        Plots the model fit for a particular localization. Note that currently, this fit is only
+        available for Gaussian refinement.
+
+        Parameters
+        ----------
+        node_idx : int
+            Index of the coordinate to plot.
```

Comment:
this still doesn't sound correct to me
```suggestion
            Index of the track node to plot (corresponding to a specific scan line).
```

Reply:
Oi. I thought I had gotten all of them. Thanks for catching that.